src/Core/Layer/LayerUpdateState.js

```diff
@@ -31,14 +54,23 @@ LayerUpdateState.prototype.canTryUpdate = function canTryUpdate(timestamp) {
         }
         case UPDATE_STATE.ERROR:
         default: {
+            const lastFailParams = this.lastFailureParams() || {};
+            const beforeLastFailParams = this.failureParams[this.failureParams.length - 2] || {};
+            // if two last Failure params aren't equivalent we can try without delay
+            if (!areEquivalentFailureParams(lastFailParams, beforeLastFailParams)) {
+                return true;
+            }
             const errorDuration = this.secondsUntilNextTry() * 1000;
             return errorDuration <= (timestamp - this.lastErrorTimestamp);
         }
     }
 };
 
 LayerUpdateState.prototype.secondsUntilNextTry = function secondsUntilNextTry() {
-    if (this.state !== UPDATE_STATE.ERROR) {
+    const lastFailParams = this.lastFailureParams() || {};
+    const beforeLastFailParams = this.failureParams[this.failureParams.length - 2] || {};
+    // if two last failure params aren't equivalent we can try without delay
```

Comment:
Your comment is describing what your doing (which is clear enough from the var/func names).
What's missing is the *why* you're doing this?